test/parallel/test-util-format.js

```diff
@@ -51,19 +51,19 @@ assert.strictEqual(util.format('%d', 42.0), '42');
 assert.strictEqual(util.format('%d', 42), '42');
 assert.strictEqual(util.format('%d', '42'), '42');
 assert.strictEqual(util.format('%d', '42.0'), '42');
-assert.strictEqual(util.format('%d', 1.5), '1.5');
-assert.strictEqual(util.format('%d', -0.5), '-0.5');
-assert.strictEqual(util.format('%d', -0.0), '-0');
-assert.strictEqual(util.format('%d', ''), '0');
+assert.strictEqual(util.format('%d', 1.5), '1');
+assert.strictEqual(util.format('%d', -0.5), '-0');
```

Comment:
NIT: in chrome `console.log('%d', '-0.5')` outputs `0`

Reply:
Sorry I'm not familiar with that acronym 😅 though this is a good point of difference between both integer formats in server and in browser (confirmed in Firefox as well).

Should this be corrected (likely not in this unit of work) or just noted?

Reply:
NIT = Nitpick (something that usually can be ignored after code review process) :)

Reply:
NIT = Nitpick (something that doesn't affect the overall PR and can, but doesn't necessarily have to be updated in order for it to be mergeable) :)

Reply:
Given that node and chrome are v8, that seems more important to align with than firefox here?

Reply:
Agreed 100%, was just noting that Firefox behaved the same as Chrome in this situation with the sign flip, so it's not specific to just Chrome/V8